tests/integration/test_convolve.py

```diff
@@ -21,31 +21,37 @@
 import cunumeric as num
 
 SHAPES = [(100,), (10, 10), (10, 10, 10)]
-
 FILTER_SHAPES = [(5,), (3, 5), (3, 5, 3)]
+MODES = ["same", "valid", "full"]
 
 
 @pytest.mark.xfail
 def test_none():
-    # Numpy raises:
-    # TypeError: unsupported operand type(s) for *: 'NoneType' and 'NoneType'
-    with pytest.raises(AttributeError):
+    expected_exc = AttributeError
+    with pytest.raises(expected_exc):
         num.convolve(None, None, mode="same")
+    with pytest.raises(expected_exc):
+        np.convolve(None, None, mode="same")
```

Comment:
But Numpy does raise a TypeError, so the `expected_exc` should be TypeError?
```
In [2]: np.convolve(None, None, mode="same")
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In [2], line 1
----> 1 np.convolve(None, None, mode="same")

File <__array_function__ internals>:180, in convolve(*args, **kwargs)

File ~/anaconda/envs/dev-311/lib/python3.11/site-packages/numpy/core/numeric.py:850, in convolve(a, v, mode)
    848 if len(v) == 0:
    849     raise ValueError('v cannot be empty')
--> 850 return multiarray.correlate(a, v[::-1], mode)

TypeError: unsupported operand type(s) for *: 'NoneType' and 'NoneType'
```
I think leaving in the comments describing the difference in xfail tests is also a good idea